tests_config/run_spec.js

```diff
@@ -29,14 +39,24 @@ function run_spec(dirname, parsers, options) {
       filename[0] !== "." &&
       filename !== "jsfmt.spec.js"
     ) {
+      if (!semver.satisfies(PYTHON_VERSION, versionRange)) {
+        // Skip each test here with the snapshot name below to avoid obsolete
+        // snapshot failures.
+        parsers.forEach(() => {
+          // eslint-disable-next-line jest/no-disabled-tests
+          test.skip(filename);
+        });
+        return;
+      }
+
       const source = read(path).replace(/\r\n/g, "\n");
 
       const mergedOptions = Object.assign({}, options, {
         parser: parsers[0]
       });
       const output = prettyprint(source, path, mergedOptions);
       test(`${filename} - ${mergedOptions.parser}-verify`, () => {
-        expect(raw(source + "~".repeat(80) + "\n" + output)).toMatchSnapshot(
+        expect(raw(source + "~".repeat(79) + "\n" + output)).toMatchSnapshot(
```

Comment:
this was a silly change, but 79 makes more sense than 80 if you know it's going to be python code formatted to 79 chars

Reply:
Should do `.repeat(mergedOptions.printWidth)`. FYI I'd like to eventually share this file from Prettier as a package.

Reply:
Fixed.

Merging these would be cool. We'd still have to wrap it for Python to do the version thing, though.